website/src/content/api-reference/commercetools-frontend-application-shell-connectors.mdx

```diff
@@ -72,6 +72,42 @@ The information about the logged in user.
 }
 ```
 
+For users who have logged-in into the Merchant Center using SSO, there will be another object property (**idTokenUserInfo**) which will contain some standard claims from the id token we get in our backend when processing the OpenId Connect (OIDC) flow with their Identify Provider.
+
+```json
```

Comment:
````suggestion
```json highlightedLines="14-22"
````

Reply:
Updated in this commit: 6898054f37f9176a6f1fadd0c0766f4d64454c6b